src/commands/local.ts

```diff
@@ -60,7 +60,7 @@ import inquirer, { Answers } from "inquirer";
 import { DEFAULT_NODE_RUNTIME } from "../models/nodeRuntime.js";
 import { getNodeModulePackageJsonLocal } from "../generateSdk/templates/packageJson.js";
 import { compileSdk } from "../generateSdk/utils/compileSdk.js";
-import { exit } from "process";
+import { cwd, exit } from "process";
```

Comment:
Looks like you are not using it.

```suggestion
import { exit } from "process";
```